src/realm-settings/RealmSettingsSection.tsx

```diff
@@ -305,6 +318,20 @@ export const RealmSettingsSection = () => {
             >
               <EventsTab />
             </Tab>
+
+            <Tab
+              id="localization"
+              eventKey="localization"
+              title={<TabTitleText>{t("localization")}</TabTitleText>}
+            >
+              <LocalizationTab
+                key={key}
+                refresh={refresh}
+                save={save}
+                reset={() => setupForm(realm!)}
+                realm={realm!}
+              />
+            </Tab>
```

Comment:
The tab will be show first when the realm is not yet loaded, it's better to show it when the data is there.
```suggestion
            {realm && (
              <Tab
                id="localization"
                eventKey="localization"
                title={<TabTitleText>{t("localization")}</TabTitleText>}
              >
                <LocalizationTab
                  key={key}
                  refresh={refresh}
                  save={save}
                  reset={() => setupForm(realm)}
                  realm={realm}
                />
              </Tab>
            )}

```